cpp/tests/utilities/column_utilities.cu

```diff
@@ -308,9 +304,12 @@ struct column_comparator_impl<list_view, check_exact_equality> {
         return lhs_offsets[index] == rhs_offsets[index] ? false : true;
       });
 
-    // shrink back down
-    differences.resize(thrust::distance(differences.begin(), diff_iter));
-    print_differences(differences, lhs, rhs, print_all_differences, depth);
+    differences.resize(thrust::distance(differences.begin(), diff_iter));  // shrink back down
+
+    if (not differences.empty()) {
+      auto const msg = differences_message(differences, lhs, rhs, print_all_differences, depth);
+      EXPECT_EQ(differences.size(), size_t{0}) << msg;
```

Comment:
Same as above

Reply:
@rgsl888prabhu The problem is that we wan't this to fail all the time. `ASSERT_FALSE` won't fail when it is false. I will look into if there is a macro for this.

Reply:
Found it. `GTEST_FAIL`. Replaced :)

Reply:
`FAIL()` can be used. At this point, EXPECT_EQ seems trivially readable.